src/VisualStudio/Core/Def/Implementation/InlineRename/InlineRenameUndoManager.cs

```diff
@@ -144,9 +144,17 @@ public void UndoTemporaryEdits(ITextBuffer subjectBuffer, bool disconnect)
 
             protected override void UndoTemporaryEdits(ITextBuffer subjectBuffer, bool disconnect, bool undoConflictResolution)
             {
-                var undoManager = this.UndoManagers[subjectBuffer].UndoManager;
-                var startRenameUndoPrimitive = this.UndoManagers[subjectBuffer].StartRenameSessionUndoPrimitive;
-                var markerPrimitive = this.UndoManagers[subjectBuffer].ConflictResolutionRenameUndoPrimitive ?? startRenameUndoPrimitive;
+                // We have reports of the BufferUndoState being unavailable here when inline rename
```

Comment:
"we have reports" -- do we have an internal repro of this? Just making sure the comment is accurate for whoever will next get the honor of seeing this.